WcaOnRails/app/views/delegate_reports/_delegate_report.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="delegate-report">
   <div>
     <strong>Date</strong>
-    <%= wca_date_range(competition.start_date, competition.end_date) %>
+    <%= wca_date_range(competition.start_date, competition.end_date, locale: :en) %>
     <% if delegate_report.schedule_url.present? %>
```

Comment:
No we must not: this template is also used for sending the mail to Delegates!

Maybe you translated labels for fields in the edit form? These are localized.